src/plugins.ts

```diff
@@ -0,0 +1,69 @@
+import Pod from './pod';
+
+export interface PluginComponent {
+  key: string;
+  name: string;
+  [x: string]: any;
```

Comment:
I think we should document all the different hooks that a plugin can have, e.g.:

```
onRendererCreate?: (renderer: Renderer) => void;
```

Reply:
That is a good point, we could add the specific handlers here which would help to track which triggers are in use.